### PR TITLE
fix `WALL_OF_SHAME` on modrinth

### PR DESCRIPTION
**Pull request by AnAwesomGuy**
that's all! :)

### CODE REVIEW

**Review by MichaelHillcox — APPROVED** [review]
Looks good to me. Although, do you happen to have a list of what modrinth supports for it's `game_version` argument. I can't seem to find anything.

**Comment by AnAwesomGuy** [comment]
> Looks good to me. Although, do you happen to have a list of what modrinth supports for it's `game_version` argument. I can't seem to find anything.

we iterate over Mojang's `https://piston-meta.mojang.com/mc/game/version_manifest_v2.json` in `MinecraftApi`, and get the versions from there. however, modrinth doesn't accept versions with spaces in their name, so this PR replaces the ones with spaces with the [ones modrinth uses](https://github.com/modrinth/labrinth/blob/ae1c5342f2017c1c93008d1e87f1a29549dca92f/src/scheduler.rs#L112).

File: src/test/kotlin/me/modmuss50/mpp/test/modrinth/ModrinthTest.kt [hunk 1]
```diff
@@ -229,7 +229,7 @@ class ModrinthTest : IntegrationTest {
                     projectId = "12345678"
 
                     minecraftVersionRange {
-                        start = "1.19.4"
+                        start = "1.13.1" // test WALL_OF_SHAME
```
Comment (modmuss50) [inline src/test/kotlin/me/modmuss50/mpp/test/modrinth/ModrinthTest.kt:232]: This should then test that the correct versions versions were submited bellow. It needs to check the new min version, and now one of the special versions.

**Review by modmuss50 — COMMENTED** [review]
Hi, sorry I tottaly missed this PR. The change looks good, the test just need improving to show that its working as expected.

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**